ng-dev/pr/common/checkout-pr.ts

```diff
@@ -109,6 +113,13 @@ export async function checkOutPullRequestLocally(
   try {
     // Fetch the branch at the commit of the PR, and check it out in a detached state.
     info(`Checking out PR #${prNumber} from ${fullHeadRef}`);
+    info(`To push to this PR branch:`);
+    info(
+      `  git push ${GITHUB_SSH_PREFIX}:${pr.headRef.repository.nameWithOwner} HEAD:${headRefName} -f`,
+    );
+    info(
+      `  git push ${GITHUB_URL_PREFIX}:${pr.headRef.repository.nameWithOwner} HEAD:${headRefName} -f`,
```

Comment:
Does this URL work? I was assuming it should be `https://github.com/{nameWithOwner}<.git>` and optionally `.git` suffix. The colon seems to cause an invalid URL

Reply:
Correct, I think this was a result of copy and paste from the ssh url above.